src/App.tsx

```diff
@@ -1,24 +1,109 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { deleteTodo, getTodos, createTodo } from './api/todos';
+import { FilterType } from './types/Filter';
+import { ErrorMessage } from './types/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 35;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<FilterType>(FilterType.ALL);
+  const [errorMessage, setErrorMessage] = useState(ErrorMessage.NONE);
+
+  const closeErrorMsg = () => {
+    setErrorMessage(ErrorMessage.NONE);
+  };
+
+  const neError = (errorMes: ErrorMessage) => {
+    setErrorMessage(errorMes);
+    setTimeout(() => {
+      closeErrorMsg();
+    }, 3000);
+  };
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        neError(ErrorMessage.CANNOT_LOAD_TODOS);
+      });
+  }, []);
```

Comment:
fix this warning
![Screenshot from 2024-01-28 15-58-04](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/33029864/1657c18a-6ddc-4676-9936-d17bc7f5b6ab)